web/modules/webspark/webspark_blocks/webspark_blocks.install

```diff
@@ -277,6 +277,14 @@ function webspark_blocks_update_9027(&$sandbox) {
   // _webspark_blocks_revert_module_config();
 }
 
+/**
+ * Uninstall CKEditor4 and FakeObjects modules.
+ */
+function webspark_blocks_update_10000(&$sandbox) {
```

Comment:
@tbutterf `fakeobjects` was already removed in #255 in`webspark` profile.  Let's keep module install/uninstalls centralized to the profile unless there's a good reason for it to happen in another module. Let's move the uninstall of the `ckeditor` module over there, please.

Reply:
I have removed it.